### PR TITLE
Display app icons in the status view

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- connect to a model
- open the status view
- check that there are icons next to the app names

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8030/

File: jujugui/static/gui/src/app/components/status/status.js [hunk 1]
```diff
@@ -298,6 +298,7 @@ class Status extends React.Component {
     A predicate function that can be used to filter units so that uncommitted
     and unplaced units are excluded.
     @param {Object} unit A unit as included in the database.
+    @returns {Boolean} Whether the unit is real or not.
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/status/status.js]: Thanks!

**Comment by frankban** [comment]
QA ok, very nice!

**Comment by hatched** [comment]
Thanks @huwshimi 

:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**